services/Skyra.Cdn/Controllers/CdnController.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Skyra.Database;
+using Skyra.Database.Models.Entities;
+
+namespace Skyra.Cdn.Controllers
+{
+	[ApiController]
+	[Route("[controller]")]
+	public class CdnController : ControllerBase
+	{
+		private readonly IDatabase _database;
+
+		public CdnController(IDatabase database)
+		{
+			_database = database;
+		}
+
+		[HttpGet("{id}")]
+		[ResponseCache(Duration = 60 * 60, Location = ResponseCacheLocation.Client, NoStore = false)]
```

Comment:
Pls no magic numbers, make these constants (what does each 60 represent?)

Reply:
60 seconds (1m) * 60, aka 60m, aka 1h, I wanted to make this more readable, but I'm unsure how, `Duration` takes an integer

Reply:
Two private const int's will do

Reply:
I can't use `TimeSpan.FromHours(1).Seconds` because it needs to be constant.

Reply:
again - two constant integers `private const int _seconds = 60; private const int _minutes = 60;`